README.md

```diff
@@ -16,12 +16,106 @@ make env
+
+Database (root)
+
+    The DATABASE values are defined to connect to cloud RDS or local DB installation.
+    These values are used by the initial Django migrations *only* to configure the application.
+
+        DATABASE_NAME: posgres database name
+        DATABASE_HOST: posgres host url 
+            - use 'localhost' for local development database
+            - use the actual tier database host URL for cloud deployment
+        DATABASE_PORT: optional posgres port - if not specified then default is '5432' 
+        DATABASE_USERNAME: postgres root user name
+        DATABASE_PASSWORD: postgres root password
+
+Database (app)
+
+    The following should be set to initial values for a new database configuration.
+    Once the database initial migration scripts are run then they need to be set to 
+    expected values. While these can be arbitrarily configurable for local database 
+    development, some example values are mentioned.
+
+    The application database ensures that application data is not stored in the postgres 
+    system database. It also enables multiple application to share a database instance.
+        APP_DATABASE_NAME:     name for the applicaiton database within postgres
+        APP_DB_OWNER_USERNAME: user name for the database owner
+        APP_DB_OWNER_PASSWORD: database owner password
+
+    The application schema ensure that applicaiton data is not stored in the default public
+    schema. The configuration assigns this schema as the primary search_path.
+        APP_SCHEMA_NAME:           name for the application schema
+        APP_SCHEMA_OWNER_USERNAME: owner user name for the application schema
+        APP_SCHEMA_OWNER_PASSWORD: schema owner password
+
+    This connection will allow maitainance of all registry entries in support of user issues.
+    It has less restricted table CRUD roles than the application user.
+        APP_ADMIN_USERNAME: user name for the connection used by the registry administration
+        APP_ADMIN_PASSWORD: administration login password
+
+    This connecton is the standard or default applicaiton user connection.
+    It has limited table CRUD roles.
+        APP_CLIENT_USERNAME: user name for the connection used by the registry users
+        APP_CLIENT_PASSWORD: user level login password
+
+## Configuring Local Database
+
+There is a sample batch script to configure a local postres instance on Windows.
+
+    migration_test.bat
+
+I simple translation to bash script will allow running it on linux.
+It is included in the project more as an example documentation than a use case.  
+The critical components of the batch script show an example of local environment variables.
+It also shows the necessary order to run Django migrations.
+
+    python -m manage migrate --database=postgres postgres
+    python -m manage migrate registry 0000
+    python -m manage migrate registry
+
+Notice that the first scripts are run while connecting for the only time with the postgres user.
```

Comment:
This is also confusing. Can you say "first script"? I assume the rest are run anytime migrations are updated?

Reply:
If you look carefully then you will notice the first execution specifies the postgres connection alias and the others need not do so because they use the default migration connection.

Reply:
Ah, maybe I misunderstand your question.

Reply:
Sorry, I misinterpreted.